src/fixed/method.rs

```diff
@@ -638,4 +749,50 @@ mod tests {
         );
         method.buffer.clear();
     }
+
+    #[test]
+    fn test_old_kar_order() {
+        let mut method = FixedMethod::default();
+        let mut config = get_fixed_method_defaults();
+        config.set_fixed_old_kar_order(true);
+
+        method.buffer = "".to_string();
+        method.process_key_value("ৈ", &config);
+        method.process_key_value("ক", &config);
+        assert_eq!(method.buffer, "কৈ".to_string());
+
+        method.buffer = "তে".to_string();
+        method.process_key_value("া", &config);
+        assert_eq!(method.buffer, "তো".to_string());
+
+        method.buffer = "সি".to_string();
+        method.process_key_value(&B_HASANTA.to_string(), &config);
+        method.process_key_value("ক", &config);
+        assert_eq!(method.buffer, "স্কি".to_string());
+
+        method.buffer = "তি".to_string();
+        method.process_key_value("্র", &config);
+        assert_eq!(method.buffer, "ত্রি".to_string());
+
+        // Vowel making with Hasanta
```

Comment:
Add a test case for making a vowel with Hasanta and left standing kar(`ি`) also.